### PR TITLE
Meta: Improve look and copy of Disable Features options

### PR DESCRIPTION
**Pull request by bfred-it**
new disabled copy

Dropped "feature names might change" because [we have migrations](https://github.com/sindresorhus/refined-github/pull/921/commits/e3a83a446fd137ef51bd56ee0dcaceb3aae3f2a5), so we can take care of that.

### CODE REVIEW

**Comment by bfred-it** [comment]
A possible ""improvement"" would be to stick the section in a `details`. A bit annoying for those who use it though.

**Comment by sindresorhus** [comment]
> A possible ""improvement"" would be to stick the section in a details. A bit annoying for those who use it though.

I think we can keep it there until we add more stuff and need the space.

**Comment by bfred-it** [comment]
“Like it is” in this PR or without it?

**Comment by sindresorhus** [comment]
This PR

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

File: source/options.html [hunk 1]
```diff
@@ -15,16 +15,16 @@ <h2>GitHub Enterprise</h2>
 
 	<h2>Disable features</h2>
 	<p>
-		<strong>Notice:</strong> All JavaScript-based features can be disabled here, however the site might not work correctly since CSS changes are still applied. This is mostly untested and feature names might change.
+		List the features to disable, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">filename</a>. Enable logging to show in the console what features are enabled on each page.
 	</p>
-	<p>
-		List the features to disable, by <a href="https://github.com/sindresorhus/refined-github/tree/master/source/features" target="_blank">filename</a>:<br>
-		<textarea name="disabledFeatures" rows="5" placeholder="For example: &#10;mark-unread &#10;hide-own-stars"></textarea>
+	<p style='font-size: 0.9em'>
```
Comment (sindresorhus) [inline source/options.html]: Should use double-quotes. And should be moved to the CSS file instead of inline?

**Pull request merged**